[2022-03-15 19:24:14,823] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-10T09:00:00+00:00 [queued]>
[2022-03-15 19:24:14,828] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-10T09:00:00+00:00 [queued]>
[2022-03-15 19:24:14,828] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:24:14,828] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-03-15 19:24:14,828] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:24:14,835] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-03-10T09:00:00+00:00
[2022-03-15 19:24:14,836] {standard_task_runner.py:54} INFO - Started process 27908 to run task
[2022-03-15 19:24:14,865] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-03-10T09:00:00+00:00', '--job_id', '71', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmp2is_nh_a']
[2022-03-15 19:24:14,865] {standard_task_runner.py:78} INFO - Job 71: Subtask twitter_aluraonline
[2022-03-15 19:24:14,879] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-10T09:00:00+00:00 [running]> on host 1a222f561522
[2022-03-15 19:24:14,908] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-03-15 19:24:14,909] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-10T09:00:00.00Z&end_time=2022-03-11T09:00:00.00Z
[2022-03-15 19:24:14,929] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 19:24:15,201] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-10T09:00:00.00Z&end_time=2022-03-11T09:00:00.00Z&next_token=b26v89c19zqg8o3fpyqm2xgh1ldlqiblquxufxgf9oubh
[2022-03-15 19:24:15,201] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 19:24:15,502] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-10T09:00:00.00Z&end_time=2022-03-11T09:00:00.00Z&next_token=b26v89c19zqg8o3fpyqlob3guz9otf2oiqqtuo1i9fczh
[2022-03-15 19:24:15,503] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 19:24:15,750] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220310T090000, start_date=20220315T192414, end_date=20220315T192415
[2022-03-15 19:24:19,821] {local_task_job.py:102} INFO - Task exited with return code 0
[2022-03-15 21:24:59,446] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-10T09:00:00+00:00 [queued]>
[2022-03-15 21:24:59,451] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-10T09:00:00+00:00 [queued]>
[2022-03-15 21:24:59,451] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 21:24:59,451] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-03-15 21:24:59,451] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 21:24:59,457] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-03-10T09:00:00+00:00
[2022-03-15 21:24:59,459] {standard_task_runner.py:54} INFO - Started process 2741 to run task
[2022-03-15 21:24:59,490] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-03-10T09:00:00+00:00', '--job_id', '72', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmphqx2tpea']
[2022-03-15 21:24:59,490] {standard_task_runner.py:78} INFO - Job 72: Subtask twitter_aluraonline
[2022-03-15 21:24:59,506] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-10T09:00:00+00:00 [running]> on host 1a222f561522
[2022-03-15 21:24:59,537] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-03-15 21:24:59,538] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-10T09:00:00.00Z&end_time=2022-03-11T09:00:00.00Z
[2022-03-15 21:24:59,559] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 21:24:59,814] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-10T09:00:00.00Z&end_time=2022-03-11T09:00:00.00Z&next_token=b26v89c19zqg8o3fpyqm2xgh1ldlqiblquxufxgf9oubh
[2022-03-15 21:24:59,815] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 21:25:00,087] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-10T09:00:00.00Z&end_time=2022-03-11T09:00:00.00Z&next_token=b26v89c19zqg8o3fpyqlob3guz9otf2oiqqtuo1i9fczh
[2022-03-15 21:25:00,087] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 21:25:00,326] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220310T090000, start_date=20220315T212459, end_date=20220315T212500
[2022-03-15 21:25:04,442] {local_task_job.py:102} INFO - Task exited with return code 0
